### PR TITLE
Add auto mtls in annoucing doc for 1.4.

### PR DESCRIPTION
**Pull request by incfly**

### CODE REVIEW

**Comment by incfly** [comment]
@istio/wg-docs-maintainers any of you can take a look? adding a brief intro for the added feature in 1.4 on annoucing page for visibility.

File: content/en/news/releases/1.4.x/announcing-1.4/_index.md [hunk 1]
```diff
@@ -36,6 +36,15 @@ The authorization policy model is now in Beta with the introduction of the
 focuses on simplification and flexibility. This will also replace the old
 [`v1alpha1` RBAC policy](/docs/reference/config/security/istio.rbac.v1alpha1/).
 
+## Automatic mutual TLS
+
+We added [automatic mutual TLS support](/docs/tasks/security/authentication/auto-mtls/). It allows
+you to adopt mutual TLS without configuring destination rule. Istio automatically configures
```
Comment (geeknoid) [inline content/en/news/releases/1.4.x/announcing-1.4/_index.md]:
```suggestion
you to adopt mutual TLS without needing to configure destination rules. Istio automatically programs
```

File: content/en/news/releases/1.4.x/announcing-1.4/_index.md [hunk 2]
```diff
@@ -36,6 +36,15 @@ The authorization policy model is now in Beta with the introduction of the
...
+client sidecar proxy to send mutual TLS to server endpoints those are able to accept mutual TLS.
```
Comment (geeknoid) [inline content/en/news/releases/1.4.x/announcing-1.4/_index.md]:
```suggestion
client sidecar proxies to send mutual TLS to server endpoints that are able to accept mutual TLS.
```

File: content/en/news/releases/1.4.x/announcing-1.4/_index.md [hunk 3]
```diff
@@ -36,6 +36,15 @@ The authorization policy model is now in Beta with the introduction of the
...
+
+Currently this feature requires to explicitly enabled, and we plan to enable it by default in
```
Comment (geeknoid) [inline content/en/news/releases/1.4.x/announcing-1.4/_index.md]:
```suggestion
Currently this feature must be explicitly enabled, but we plan to enable it by default in a
```

**Pull request merged**